src/azul/service/__init__.py

```diff
@@ -44,23 +48,31 @@ def reify(self, *, explicit_only: bool) -> FiltersJSON:
     def to_json(self):
         return {
             'explicit': self.explicit,
-            'source_ids': sorted(self.source_ids)
+            'source_ids': sorted(self.source_ids),
+            'source_id_facet': self.source_id_facet
```

Comment:
Ideally, we wouldn't be passing around pieces of plugin configuration that never changes during the run time. Feel free to discuss in PL if you see now way to address this.

Reply:
Bumping review count for now but may undo depending on outcome of PL discussion.

Reply:
Decided at parking lot to pass service config as parameter to `Filters.reify` rather than add it as an attribute.